src/routes/console/project-[project]/overview/platforms/wizard/android/step2.svelte

```diff
@@ -11,8 +11,8 @@
 
 <WizardStep>
     <svelte:fragment slot="title">Get the SDK</svelte:fragment>
-    <p>First, add this to your root level build.gradle file:</p>
+    <p>First, add this to your root level <code class="inline-code">build.gradle</code> file.</p>
```

Comment:
@TGlide Can you find someone who's familiar with the Android ecosystem to see if the two sentences make sense?

All this talk about different grade files is getting a little confusing.

Reply:
@lohanidamodar maybe?

Reply:
or @abnegate Heehee